consensus/polybft/consensus_runtime.go

```diff
@@ -157,7 +171,7 @@ func (c *consensusRuntime) close() {
 func (c *consensusRuntime) initStateSyncManager(logger hcf.Logger) error {
```

Comment:
Why not reuse here the `logger` instance in `consensusRuntime`?

Reply:
No idea, just following the current pattern. We could create a ticket to clean up this part of the code.